packages/api/Dockerfile

```diff
@@ -36,6 +36,8 @@ RUN find . -name "node_modules" -exec rm -rf '{}' +
 RUN rm -rf '/app/package-lock.json'
 
 RUN sudo apt-get install nodejs
+RUN sudo npm install -g npm-check-updates
```

Comment:
Do we need check updates? Because the new update might break the code

Reply:
There was an issue after the deployment, that why we've added it. Let's keep it as of now, will remove it later if it's not needed after testing.